@xen-orchestra/web-core/lib/components/UiSeparator.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <hr class="ui-separator" />
+</template>
+
+<style lang="postcss" scoped>
+.ui-separator {
+  border: none;
+  border-top: var(--color-grey-500) solid 1px;
+  border-right: var(--color-grey-500) solid 1px;
```

Comment:
Although the values could be passed in any order, let's keep consistency across the project and use `<width> <style> <color>`. Also don't forget to use `rem` instead of `px`.
```suggestion
  border-top: 0.1rem solid var(--color-grey-500);
  border-right: 0.1rem solid var(--color-grey-500);
```